code/project.clj

```diff
@@ -95,7 +96,8 @@
                                [org.slf4j/slf4j-log4j12]
                                [com.cemerick/url]
                                [org.apache.curator/curator-test]
-                               [org.clojars.konstan/kinsky-test-jar ~kinsky-version]]
+                               [org.clojars.konstan/kinsky-test-jar ~kinsky-version]
+                               [io.swagger.parser.v3/swagger-parser "2.1.16"]]
```

Comment:
This is the official Swagger parser library, used in the tests to make sure the generated spec can be parsed correctly.